src/App.css

```diff
@@ -1,3 +1,9 @@
+.about-me{
+  background-color: #9d8d8f;
```

Comment:
why this color?

Reply:
color holder for now. I'm going to add more icons/content. I'll have to change the background color accordingly. I just wanted to see how it would render.